.github/workflows/salesloft_build.yaml

```diff
@@ -0,0 +1,53 @@
+name: Docker Build and Push
+
+on:
+  push:
+    branches:
+      - 'salesloft/*'
```

Comment:
At what point will this job run? This name doesn't match the name of the destination branch for this PR

Reply:
Ye, I didnt want it to build here, Im just making a commit here with a place to review for our changes so that we are happy with our code. I will then cherry pick this for this: https://github.com/SalesLoft/vault-secrets-operator/compare/salesloft/v0.7.0...SalesLoft:vault-secrets-operator:salesloft/v0.7.0-dev?expand=1

Reply:
Ah nice.

In that case, I wonder if the tag should be the end part of that branch name:
This line: https://github.com/SalesLoft/vault-secrets-operator/pull/3/files#diff-bab801f03e9c67157e73557d73b56f3bd18e1d281e0380d2c6e2c79e9142d256R50

Change it to match the part after `salesloft/` so that our tags contain the version number in them.